### PR TITLE
FastSpeech2,Hifi-GAN, Parallel WaveGAN, MelGAN TFLite models

### PR DESCRIPTION
**Pull request by tulasiram58827**
Any pull request you open is subject to the TensorFlow Hub Terms of Service at www.tfhub.dev/terms.

### CODE REVIEW

**Comment by tulasiram58827** [comment]
@maringeo Gentle ping

**Comment by tulasiram58827** [comment]
Thanks for the review @WGierke. Updated as per your suggestion.

File: assets/docs/tulasiram58827/parallel_wavegan/float16/1.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+@misc{yamamoto2020parallel,
+      title={Parallel WaveGAN: A fast waveform generation model based on generative adversarial networks with multi-resolution spectrogram},
+      author={Ryuichi Yamamoto and Eunwoo Song and Jae-Min Kim},
+      year={2020},
+      eprint={1910.11480},
+      archivePrefix={arXiv},
+      primaryClass={eess.AS}
+}
+```
+
```
Comment (WGierke) [inline assets/docs/tulasiram58827/parallel_wavegan/float16/1.md:35]: Nit: would you mind to remove that? Sorry for overlooking it in my first pass

**Comment by tulasiram58827** [comment]
Oh sorry, I did not notice it. Updated @WGierke

**Pull request merged**